### DISCUSSION TITLE
PatchTool 6.31 2/1 修正文本

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#295] [Announcements]
修正部分對話出現空白的問題。

※2/9更新：新增6.31維護更新後之還原檔和修改了本體檔案的config。由於沒有發現相容性問題，有先還原再更新者可正常使用原本的漢化包，無須重新下載。

檔案：

- ~~FFXIVChnTextPatch-20230201.rar：本體。一般而言，下載這個即可。~~
- FFXIVChnTextPatch-20230209.rar：本體。一般而言，下載這個即可。

選用：

- [font-custom-4k-smaller-20230131.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.0.1/font-custom-4k-smaller-20230131.rar)：高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)透過重複使用字體字型縮小檔案體積，使用Noto Sans。
- [font-custom-20230111.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.72.2/font-custom-20230111.rar)：舊高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)所有大小字體皆分開繪製，使用Noto Sans和Noto Serif。
- rawexd-zhcn-20230201.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/90 放到與exe同位置使用。
- backup-20230209.rar：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

※若不替換字體會導致遊戲內無法正常顯示部分中文字。另外，目前大廳部分中文字必須使用高畫質字體包才能正常顯示。更多資訊可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

有問題歡迎至Issues或Discussions回報。

**Comment by snowmaple5891** [comment]
![image](https://user-images.githubusercontent.com/80057212/216824092-2f1047b9-adc8-4246-8a20-0a83e08dcea9.png)
您好，我的介面中沒有出現目標語言，請問這樣要怎麼修復?

**Reply by GpointChen** [reply]
使用CSV時無須更改目標語言項

**Comment by chun1324** [comment]
![image](https://user-images.githubusercontent.com/15475075/216828035-892fa2d0-193c-4e8d-b218-44adcbe55f3c.png)
這次發佈預設替換文本改成沒有勾選, 結果產生沒有成功漢化的錯覺

**Reply by GpointChen** [reply]
沒注意到，感謝告知

**Comment by Bos-hunting** [comment]
今天有6.31的hotfix更新包，希望有相容性更新！謝謝作者

**Reply by GpointChen** [reply]
感謝提醒，看了一下沒有影響相容性的改動，只要有正常先還原再更新應該都可以正常使用。

晚點我上傳新的backup files。

**Reply by GpointChen** [reply]
已上傳。

**Reply by Bos-hunting** [reply]
感謝！

**Comment by paulyang81522** [comment]
59等任務 沉睡在禁書庫的論文 最開始有一段NPC的對話還是原文，沒有漢化到。

**Reply by paulyang81522** [reply]
![擷取](https://user-images.githubusercontent.com/125285143/218734242-7ad5f2c1-4c3f-4a78-8efe-4105ab5a1442.JPG)

**Comment by dnangel8042110** [comment]
![image](https://user-images.githubusercontent.com/8119383/219844168-182be906-a699-4b37-ace8-a63ad6ec9619.png)
第一次交納老主顧艾爾‧圖時，跳出的提示會卡在最後一行無法結束，只能重開遊戲

**Reply by dnangel8042110** [reply]
所有老主顧只要第一次對話有跳出這段都會卡住

**Reply by GpointChen** [reply]
用最新版本測試沒有出現這個問題，可以試用看看。

**Comment by LL9754** [comment]
![image](https://user-images.githubusercontent.com/125777278/219908040-3a779148-8bb4-4d20-9bf7-e82242e89f12.png)
該怎麼辦呢 好想漢化QQ

**Reply by LL9754** [reply]
![image](https://user-images.githubusercontent.com/125777278/219908157-65e7e951-3e42-4510-8e04-47267f3c8c37.png)
還會一直跳出這個

**Reply by chun1324** [reply]
首先你不應該從解壓縮軟體裡執行
再來你要選擇正確的FF14遊戲位置

**Reply by chun1324** [reply]
常見位置
C:\Program Files (x86)\SquareEnix\FINAL FANTASY XIV - A Realm Reborn
Steam版
C:\Program Files (x86)\Steam\steamapps\common\FINAL FANTASY XIV - A Realm Reborn

**Reply by LL9754** [reply]
問題解決了!! 謝謝你 熱心的朋友

**Reply by LL9754** [reply]
![image](https://user-images.githubusercontent.com/125777278/219910038-6a2c999a-99a5-4dcd-91d1-4875ea6cff11.png)
那個 = 是正常的嗎

**Reply by chun1324** [reply]
正常, 下載點那邊說明和解決方法. 不解決也能如常遊玩

**Comment by nanayaarashi** [comment]
6.3尚未有漢化字體請問是正常嗎?

**Reply by chun1324** [reply]
正常, 這個工具將陸服翻譯(6.2)帶到國際服上使用(6.31)
如果你指的是遊戲大廳缺字, 請參考這篇
https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-字體

**Comment by Zhong0514** [comment]
一樣是第一次接老主顧的時候會卡死 只能重開
![老主 (2)](https://user-images.githubusercontent.com/121940592/222974928-a4c9811b-ba8f-468e-808c-5e33c69353ae.png)

**Reply by GpointChen** [reply]
用最新版本測試沒有出現這個問題，可以試用看看。

**Comment by Bos-hunting** [comment]
今天有6.35的更新，麻煩作者了謝謝！

**Reply by GpointChen** [reply]
已更新
